### ISSUE TITLE
New Commits aren't reflected in the IUS repo

### ISSUE THREAD

**Issue by bgibson710** [iusrepo/php72u#14]
Looks like you have commits for 7.2.19 and 7.2.20 but the latest in the IUS repo seems to be 7.2.18.

**Comment by carlwgeorge** [comment]
The IUS repo is in the process of switching from the traditional mirror network to a CDN.  On the CDN, [7.2.19-1](https://repo.ius.io/7/x86_64/packages/p/php72u-common-7.2.19-1.el7.ius.x86_64.rpm) is available in the main repo, and [7.2.20-1](https://repo.ius.io/testing/7/x86_64/packages/p/php72u-common-7.2.20-1.el7.ius.x86_64.rpm) is available in the testing repo.  If you can only see 7.2.18, then you are probably still using the legacy mirror network.  The latest ius-release package changes the yum configuration to use the CDN.  We plan to complete the transition this month and promote the new ius-release package to the stable repo.  If you don't want to wait, you can enable it yourself now.

```
yum --enablerepo ius-testing update ius-release
yum --enablerepo ius-testing clean all
```

More details can be found [here](https://github.com/iusrepo/infrastructure/issues/14).

**Comment by carlwgeorge** [comment]
I forgot to mention, the CDN change was announced [here](https://github.com/iusrepo/announce/issues/13).  We use that GitHub repository to announce major changes like this and package retirements, and recommend that every IUS user subscribe to it for updates.
